common/util.go

```diff
@@ -15,6 +15,6 @@ func VerifyIsURLResolvable(url_string string) (error) {
 		return err
 	}
 
-	_, err = net.LookupIP(url.Host)
+	_, err = net.LookupIP(url.Hostname())
```

Comment:
Is there a unit test for this method we can add some cases to? If not might be a good idea to add a few tests?

Reply:
+1